test/java/seedu/addressbook/util/TestUtil.java

```diff
@@ -75,4 +80,19 @@ public static void assertTextFilesEqual(Path path1, Path path2) throws IOExcepti
         List<String> list2 = Files.readAllLines(path2, Charset.defaultCharset());
         assertEquals(String.join("\n", list1), String.join("\n", list2));
     }
+
+    /**
+     * Executes the command, and asserts the result message is as expected.
+     */
+    public static void assertCommandResult(Command command, AddressBook addressBook,
+                                           AddressBook expectedAddressBook, String expectedMessage) {
```

Comment:
@pyokagan there is a `TODO` below which asks to implement equals method in `CommandResult`, then compare the expected `CommandResult `and actual